From fe00842e1562d78f2811c8dad46125041cccab87 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Thu, 9 Jul 2015 16:18:39 +0200
Subject: [PATCH] Add suport for new libsystemd

In systemd v209, the various libraries were merged into a single
libsystemd library. When checking for systemd > 209, use the new library
name instead of the deprecated libsystemd-login.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/configure.ac
+++ b/configure.ac
@@ -380,13 +380,13 @@
 AS_IF([test "x$enable_resize_optimization" = "xyes"],
       [AC_DEFINE([USE_RESIZE_POOL], [1], [Use resize memory pool as a performance optimization])])
 
-PKG_CHECK_MODULES(SYSTEMD_LOGIN, [libsystemd-login >= 198],
+PKG_CHECK_MODULES(SYSTEMD_LOGIN, [libsystemd >= 198],
                   [have_systemd_login=yes], [have_systemd_login=no])
 AS_IF([test "x$have_systemd_login" = "xyes"],
       [AC_DEFINE([HAVE_SYSTEMD_LOGIN], [1], [Have systemd-login])])
 AM_CONDITIONAL(HAVE_SYSTEMD_LOGIN, test "x$have_systemd_login" = "xyes")
 
-PKG_CHECK_MODULES(SYSTEMD_LOGIN_209, [libsystemd-login >= 209],
+PKG_CHECK_MODULES(SYSTEMD_LOGIN_209, [libsystemd >= 209],
                   [have_systemd_login_209=yes], [have_systemd_login_209=no])
 AS_IF([test "x$have_systemd_login_209" = "xyes"],
       [AC_DEFINE([HAVE_SYSTEMD_LOGIN_209], [1], [Have systemd-login >= 209])])
